setup.py

```diff
@@ -14,6 +17,8 @@
     description="Framework for machine learning on source code. "
                 "Provides API and tools to train and use models based "
                 "on source code features extracted from Babelfish's UASTs.",
+    long_description=long_description,
```

Comment:
@zurk Can you please add this nice feature to all our Python projects.

Reply:
sure